calcom/cal.com
# [CAL-1142] CSP Violation[Report Only]: PhoneInput component loads flags from different domain

## What does this PR do?
Adds country-flag-icons from a third party domain to our public folder to
be used by react-phone-number-input. This is to resolve a CSP warning we get in
browser whenever country flag svgs are loaded from external domain for phone-number-input

Fixes #1142


**Environment**: Staging(main branch) / Production

## Type of change



- Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?



Visit all places where phone number input is used and make sure that the relevant flag shows up for all countries
selected. The places tested include
- [ ] Workflow - send sms to a particular number
- [ ] Booking - attendee phone input
- [ ] Add Action dialog for workflow - send sms to a particular number

## Checklist



- ~~I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)~~
- ~~My code doesn't follow the style guidelines of this project~~
- ~~I haven't performed a self-review of my own code and corrected any misspellings~~
- ~~I haven't commented my code, particularly in hard-to-understand areas~~
- ~~I haven't checked if my PR needs changes to the documentation~~
- ~~I haven't checked if my changes generate no new warnings~~
- ~~I haven't added tests that prove my fix is effective or that my feature works~~
- ~~I haven't checked if new and existing unit tests pass locally with my changes~~

/claim #7300
